src/config/sign-up-inputs.config.js

```diff
@@ -0,0 +1,25 @@
+import CONSTANTS from 'constants'
```

Comment:
Isn't `node:constants` module deprecated? How does this approach compares to environment vars based configuration?

Reply:
I think the syntax `import CONSTANTS from 'constants'` is a bit confusing at first. Because that is what `NPM` modules look like. In this case, it is not an NPM/node package.

Here, I'm not using the node module constants. Probably, you're referring to [this package here.](https://www.npmjs.com/package/constants)

The file you've mentioned above exists at the directory below:

```
cd ./src/constants/index.js
```

I created it myself to save frontend-related variables.

**About the files naming convention**, I use Webpack `resolve` and `alias` properties to save me time from typing the absolute path of frequently used files. Below is a snippet of the implementation of aliases from `webpack.config.js`:

```
  resolve: {
    extensions: ['.js', '.jsx', '.css', '.json'],
    alias: {
      api: path.resolve(__dirname, 'src/api/'),
      components: path.resolve(__dirname, 'src/components/'),
      config: path.resolve(__dirname, 'src/config/'),
      constants: path.resolve(__dirname, 'src/constants/'),
      styles: path.resolve(__dirname, 'src/styles/'),
      pages: path.resolve(__dirname, 'src/pages/'),
      utility: path.resolve(__dirname, 'src/utility/'),
    },
  },
````

These are my conventions and how I usually work.